README.md

```diff
@@ -503,15 +503,18 @@ trainer = Trainer("experiment_name")
 model = ...
 
 training_params = { ...  # Your training params
-                   "sg_logger": "dagshub_sg_logger",  # DagsHub Logger, see class super_gradients.common.sg_loggers.dagshub_sg_logger.DagsHubSGLogger for details
-                   "sg_logger_params":  # Params that will be passes to __init__ of the logger super_gradients.common.sg_loggers.dagshub_sg_logger.DagsHubSGLogger
-                     {
-                       "dagshub_repository": "<REPO_OWNER>/<REPO_NAME>", # Optional: Your DagsHub project name, consisting of the owner name, followed by '/', and the repo name. If this is left empty, you'll be prompted in your run to fill it in manually.
-                       "log_mlflow_only": False, # Optional: Change to true to bypass logging to DVC, and log all artifacts only to MLflow  
-                       "save_checkpoints_remote": True,
-                       "save_tensorboard_remote": True,
-                       "save_logs_remote": True,
-                     }
+                   "sg_logger": {"dagshub_sg_logger":
```

Comment:
Should we also introduce logger names consistent with class names and deprecate old snake_case names? E.g "DagsHubSGLogger" instead of "dagshub_sg_logger"?

Reply:
I was thinking about that too actually.
Should I? Then since we don't want to break (not actually breaking since this is a new format) we would also deprecate the old way?
Or should we just leave both options ?

Reply:
I think it would be great, add the class names, and deprecate the old names.